src/components/NFTPage/CollectorList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <DropDownList
+    :class="[
+      'w-full',
+      rootClass
+    ]"
+    :title="$t('nft_details_page_title_collector')"
+  >
+    <template #titleIcon>
+      <slot name="titleIcon" />
+    </template>
+    <template #content>
+      <div class="flex flex-col my-[12px]">
+        <div v-if="ownerCount">
+          <div v-for="o in ownerList" :key="o">
+            <div class="flex items-center justify-between">
+              <LinkV2 :to="`/${o}`">{{ displayNameList[o] || o | ellipsis }}</LinkV2>
+              <Label preset="p6">{{ collectorList[o].length }}</Label>
```

Comment:
Suggest to create a computed property e.g. `populatedCollectors` that loop through ownerList array by `array.map` and add a `displayName` key and `collectedCount` key to ownerList array item then return the populated ownerList, so that no need to pass displayNameList & collectorList as props